mobile-controller/public/styles.css

```diff
@@ -59,7 +63,7 @@ html{
   position:absolute;
   top:170px;
   left:120px;
-  animation: Peww 1.1s infinite;
+  /* animation: Peww 1.1s infinite; */
```

Comment:
Yessir, I left them in purpose to keep tracking of which animation is referenced where, because the naming convention `item#2 { animation: Peww }`, `item#4 { animation: Pewww2 }` can be confusing. 

However, I just noticed now that a better approach is to keep these animations referenced on CSS via a custom class, and just toggle the class on JS, instead of toggle the animations.

Reply:
nice! Will you do this change?

Reply:
Yessir! Done